### DISCUSSION TITLE
'pytransform3d.visualizer' has no attribute 'Figure'

### DISCUSSION THREAD

**Discussion by sdbhosale** [dfki-ric/pytransform3d#281] [Q&A]
I installed the pytranform3d in my conda env and was trying to run the example notebook on `vis_basis.ipynb` from the [website](url). However I am getting 
`AttributeError: module 'pytransform3d.visualizer' has no attribute 'Figure'`

Any reason this would happen? Let me know if I am missing anything? Thank you in advance.

**Comment by AlexanderFabisch** [comment]
Hi, the 3D visualizer requires an optional dependency: open3d. Please let me know if installing it solves the problem.
